content/en/real_user_monitoring/dashboards/_index.md

```diff
@@ -2,24 +2,49 @@
 title: RUM Dashboards
 kind: documentation
 further_reading:
-- link: "/real_user_monitoring/installation/advanced_configuration"
-  tag: "Documentation"
-  text: "Advanced configuration for RUM data collection"
-- link: "/real_user_monitoring/explorer"
-  tag: "Documentation"
-  text: "Explore your views within Datadog"
+    - link: '/real_user_monitoring/installation/advanced_configuration'
+      tag: 'Documentation'
+      text: 'Advanced configuration for RUM data collection'
+    - link: '/real_user_monitoring/explorer'
+      tag: 'Documentation'
+      text: 'Explore your views within Datadog'
 ---
 
-{{< img src="real_user_monitoring/dashboards/rum_dashboard.png" alt="RUM Dashboard"  >}}
+When you [create a RUM application][1] dashboards are created within Datadog to analyze all the [data collected][2]. RUM dashboards can be found in the dashboards list and have the Datadog logo:
 
-When you [create a RUM application][1] a dashboard is created within Datadog to analyze all the [data collected][2].
-To access it, go in your [RUM application page][3] and select click on the **Dashboard** link associated with your application:
+{{< img src="real_user_monitoring/dashboards/rum_dashboard_in_dashlist.png" alt="RUM Dashboard in dash list" >}}
 
-{{< img src="real_user_monitoring/dashboards/application_page.png" alt="Application page"  >}}
+Another way to access them is to go in your [RUM application page][3] and click on the **Dashboard** links associated with your application:
```

Comment:
You can also access these dashboards through your [RUM application page][3]. Click on the **Dashboard** links associated with your application: